### PR TITLE
Add DevBcn to front page

### PR DESCRIPTION
**Pull request by jonathanvila**
Signed-off-by: Jonathan Vila <jonathan.vila@gmail.com>

### CODE REVIEW

File: 2022/_includes/homepage/slides.html [hunk 1]
```diff
@@ -68,6 +68,29 @@
...
+                                    <div class="counter-box-general-content-date" style="background:rgba(5, 194, 252, 0.7);">
+                                        <div class="counter-box-general-content-date-left">
+                                            <div class="counter-box-general-content-date-left-left"></div>
+                                            <div class="counter-box-general-content-date-left-top"></div>
+                                            <div class="counter-box-general-content-date-left-bottom"></div>
+                                        </div>
+                                        <div class="counter-box-general-content-date-center">
+                                            <div class="counter-box-general-content-date-days" style="text-shadow: 2px -2px white; color:black">DevBcn</div>
+                                            <div class="counter-box-general-content-date-monthyear" style="text-shadow: 2px -2px white;">July 3rd to 5th 2023</div>
+                                            <div class="counter-box-general-content-date-monthyear" style="text-shadow: 2px -2px white;">www.devbcn.com</div>
```
Comment (anyulled) [inline 2022/_includes/homepage/slides.html:84]:
```suggestion
                                            <div class="counter-box-general-content-date-monthyear" style="text-shadow: 2px -2px white;"><a href="https://www.devbcn.com" target="_blank" rel="noreferrer">devBcn</a></div>
```

**Review by anyulled — APPROVED** [review]

File: 2022/_includes/homepage/slides.html (hunk 1 above)
Comment (icougil) [reply 2022/_includes/homepage/slides.html:84]: good point Anyul.
I think it makes totally sense 👍

Comment (jonathanvila) [reply 2022/_includes/homepage/slides.html:84]: I assume you both know what a DRAFT PR is.
At the moment there's no point on reviewing it.

**Comment by jonathanvila** [comment]
Promoted to Ready To Review

**Comment by anyulled** [comment]
It doesn't look centered on my screen.

image

I believe it could be fixed if we increase the width on the counter-box-general-content-wrapper CSS class.

File: 2022/_includes/homepage/slides.html [hunk 2]
```diff
@@ -30,78 +30,74 @@
...
-                                        });
-                                    </script>
--->
+                                                </div>   
+                                            </td>
+                                            <td style="vertical-align: middle;">
+                                                <div class="counter-box-general-content-date" >
+                                                    <div class="counter-box-general-content-date-center">
+                                                        <div class="counter-box-general-content-date-monthyear" style="color: white">
+                                                        Rebranded to...&nbsp;&nbsp;
```
Comment (abelsromero) [inline 2022/_includes/homepage/slides.html:80]: If you care `Rebranded` has a negative connotation for some ppl, it's a marketing term after all. Why not simply "We have evolved, meet us next year at..."

**Comment by icougil** [comment]
> It doesn't look centered on my screen.

the same for me

**Review by anyulled — APPROVED** [review]

**Pull request merged**